src/MagnumPlugins/AnyAudioImporter/AnyImporter.cpp

```diff
@@ -53,6 +53,8 @@ void AnyImporter::doOpenFile(const std::string& filename) {
         plugin = "VorbisAudioImporter";
     else if(Utility::String::endsWith(filename, ".wav"))
         plugin = "WavAudioImporter";
+    else if(Utility::String::endsWith(filename, ".flac"))
+	plugin = "FlacAudioImporter";
```

Comment:
@LB-- Looks like that happened in a few locations, I'll patch it shortly. My nano instance isn't properly configured yet, and my eyes aren't what they used to be!